### PR TITLE
[Python] Initial draft of "lists" concept exercise (Issue #859)

### PR DESCRIPTION
**Pull request by itamargal**
Added the following files:

- .docs/instructions.md
- .meta/example.py
- lists.py
- lists_test.py

### CODE REVIEW

File: languages/python/exercises/concept/lists/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+## Instructions
+
+As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to manipulate. In this exercise we'll use lists to help Elyse perform her card tricks.
+
+[//]: # (Creating Lists)
```
Comment (SleeplessByte) [inline languages/python/exercises/concept/lists/.docs/instructions.md:5]: What does this do :O

Comment (BethanyG) [reply languages/python/exercises/concept/lists/.docs/instructions.md:5]: Looks like it makes a numbered list but at header size!  At least that's what appears in the doc preview.

File: languages/python/exercises/concept/lists/lists.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
+def to_list(arg1, arg2, arg3, arg4, arg5):
```
Comment (BethanyG) [inline languages/python/exercises/concept/lists/lists.py:1]: I might push the conceit of the  exercise narrative here a bit and use card_1 .... card_5 for the argument names.

File: languages/python/exercises/concept/lists/.docs/instructions.md (hunk 1 above)
Comment (BethanyG) [reply languages/python/exercises/concept/lists/.docs/instructions.md:5]: I love the narrative.  Great idea to use a "hand" & magic tricks!

File: languages/python/exercises/concept/lists/lists.py (hunk 2 above)
Comment (BethanyG) [inline languages/python/exercises/concept/lists/lists.py:1]: As with the comments on the `example.py` file, suggest renaming the functions and the arguments to go with the story and/or avoid referencing types and having to use a leading underscore.

File: languages/python/exercises/concept/lists/.docs/instructions.md (hunk 1 above)
Comment (BethanyG) [reply languages/python/exercises/concept/lists/.docs/instructions.md:5]: We'll want to add in relevant links to documentation on `lists` -- but we can do that later (_I can help with that_).

**Comment by BethanyG** [comment]
Closing this in favor of https://github.com/exercism/v3/pull/1932, which is the same but with `introduction.md` formatted through prettier.

**Pull request closed without merging**